R/known-sources.R

```diff
@@ -104,6 +104,40 @@ taxi_schema <- function() {
 }
 
 known_datasets <- list(
+  taxi_2013 = list(
+    url = "https://archive.org/download/nycTaxiTripData2013/trip_fare.7z",
+    download = function(path) {
+      archive_path <- file.path(dirname(path), "trip_fare.7z")
+      if (!file.exists(archive_path)) {
+        download.file(
+          "https://archive.org/download/nycTaxiTripData2013/trip_fare.7z",
+          archive_path,
+          mode = 'wb',
+          method = 'wget'
```

Comment:
We could do `method = "curl"`, but `wget` is really the more appropriate tool here, because it will pick up where it left off if it fails mid-download on a file (which it did for me), and this is one really big file, so retries that start over could potentially add hours. Mac install is just `brew install wget`. There's a Windows port too.

It's really a tradeoff between "just working" and robustness. I figured this package won't be widely used, so went with the latter.

Reply:
Added it to the README